New pages: HTMLMeterElement properties (#35638)
mdn/content: files/en-us/web/api/htmlmeterelement/optimum/index.md

---
title: "HTMLMeterElement: optimum property"
short-title: optimum
slug: Web/API/HTMLMeterElement/optimum
page-type: web-api-instance-property
browser-compat: api.HTMLMeterElement.optimum
---

{{ APIRef("HTML DOM") }}

> The **`optimum`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the optimum boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`optimum`](/en-US/docs/Web/HTML/Element/meter#optimum) attribute, or the midpoint between `min` and `max` values if not defined. The value of `optimum` is clamped by the `min` and `max` values.

Reviewer: ```suggestion
The **`optimum`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the optimum boundary of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`optimum`](/en-US/docs/Web/HTML/Element/meter#optimum) attribute, or the midpoint between `min` and `max` values if not defined. The value of `optimum` is clamped by the `min` and `max` values.
```

This property can also be set directly, for example to set a default value based on some condition.

## Value

> A number. The value of `optimum`, or the midpoint between {{DOMxRef("HTMLMeterElement.min")}} and {{DOMxRef("HTMLMeterElement.max")}} if not defined.

Reviewer: ```suggestion
A number. Defaults to the midpoint between {{DOMxRef("HTMLMeterElement.min")}} and {{DOMxRef("HTMLMeterElement.max")}} if not defined.
```
